mypyc/build.py

```diff
@@ -325,7 +328,8 @@ def mypycify(paths: List[str],
              opt_level: str = '3',
              multi_file: bool = False,
```

Comment:
Let's make `multi_file` and `verbose` part of `CompilerOptions`, while we're at it.